apollo-server/v3/integration-testing/src/main.ts

```diff
@@ -1,7 +1,3 @@
 import { createApolloServer } from './server';
 
-const createServer = async () => {
-  await createApolloServer();
-};
-
-createServer();
+createApolloServer();
```

Comment:
tbh this isn't necessarily the best demo for how to do a real main, because any errors thrown from createApolloServer will turn into "unhandled rejections". it's better to do `await createApolloServer()` but I think the particular setup you have here doesn't allow top-level await (notably it doesn't seem to work with the ts-node program).  but this is an integration testing demo I guess, and people can look at another demo like my subscriptions one to see how to get top-level await working.